src/components/VmDetails/cards/OverviewCard/index.js

```diff
@@ -88,16 +111,17 @@ class OverviewCard extends React.Component {
       this.props.onEditChange(true, true)
     }
 
+    const newState = { isDirty: true }
```

Comment:
I don't see `newState` used elsewhere.  Is it necessary or can the `isDirty: true` just be put back in the `setState` call below?

Reply:
Actually `isDirty: true` really can be putted in `setState`, but `newState` is used for adding name error.

Reply:
Ah ok, I didn't notice lines 125-129.  All good.